app/assets/stylesheets/katello/katello.scss

```diff
@@ -8,12 +8,12 @@
 @import "katello/widgets/tabs";
 @import "katello/katello_sprites";
 
-@import "katello/jquery-ui-1.8.11.custom.css";
-@import "katello/jquery.loadmask.css";
-@import "katello/jquery.jnotify.css";
-@import "katello/jquery.treeTable.css";
-@import "katello/jquery.jscrollpane.css";
-@import "katello/ui.spinner.css";
+@import "jquery-ui-1.8.11.custom.css";
+@import "jquery.loadmask.css";
+@import "jquery.jnotify.css";
```

Comment:
@ohadlevy link? sample page to test that on?
@ehelms thoughts?

I won't merge until I hear back from both of you. Thanks!